manifests/machineconfigcontroller/deployment.yaml

```diff
@@ -27,31 +27,26 @@ spec:
             cpu: 20m
             memory: 50Mi
         terminationMessagePolicy: FallbackToLogsOnError
-      - name: oauth-proxy
-        image: {{.Images.OauthProxy}}
+      - name: kube-rbac-proxy
+        image: {{.Images.KubeRbacProxy}}
         ports:
         - containerPort: 9001
           name: metrics
           protocol: TCP
         args:
-        - --https-address=:9001
-        - --provider=openshift
-        - --openshift-service-account=machine-config-controller
+        - --secure-listen-address=0.0.0.0:9001
+        - --config-file=/etc/kube-rbac-proxy/config-file.yaml
         - --upstream=http://127.0.0.1:8797
-        - --tls-cert=/etc/tls/private/tls.crt
-        - --tls-key=/etc/tls/private/tls.key
-        - --cookie-secret-file=/etc/tls/cookie-secret/cookie-secret
-        - '--openshift-sar={"resource": "namespaces", "verb": "get"}'
-        - '--openshift-delegate-urls={"/": {"resource": "namespaces", "verb": "get"}}'
+        - --logtostderr=true
+        - --tls-cert-file=/etc/tls/private/tls.crt
+        - --tls-private-key-file=/etc/tls/private/tls.key
         resources:
           requests:
             cpu: 20m
             memory: 50Mi
         volumeMounts:
-        - mountPath: /etc/tls/private
-          name: proxy-tls
```

Comment:
I think you still need to mount the the TLS certificate here (you left it in for the daemon, assuming this was an accident)  

I'll leave the "should these all have their own certificate, or is it cool that they're sharing the same one" question for a later date. :smile: